torch/quantization/fx/quantize.py

```diff
@@ -999,6 +999,34 @@ def load_arg(a):
         quantized = QuantizedGraphModule(quantized_root, folded_graph)
         return quantized
 
+    def _fold_quant_dequant(self, quantized: QuantizedGraphModule) -> QuantizedGraphModule:
+        """ If quantize op is followed by a dequantize, we fold the ops together and remove the dequant.
+            In the case where the only consumer of quantize_per_tensor is a dequant op, we erase both
+            nodes from the graph, along with the qparams associated with quantize op.
+        """
+        def _is_dequant_op(node) -> bool:
+            if node.op == 'call_method' and node.target == "dequantize":
+                return True
+            else:
+                return False
```

Comment:
would it be simple to inline it?

```
is_dequant = node.op == 'call_method' and node.target == "dequantize"
if is_dequant:
    ... do stuff ...
```

Asking because this file has a history of functions which are local to parent functions, and in those cases it's sometimes unclear which variables are passed in via args and which are from the parent function's scope.  If we need this to be a func, maybe we can move it to be a standalone function to make it easy to understand that all variables are passed in?

Reply:
makes sense to inline it. I thought there would be more to this function when I first wrote it.